src/arrayMethodSort.js

```diff
@@ -2,11 +2,20 @@
 
 /**
  * Implement method Sort
- */
+//  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction =
+  (a, b) => String(a) >= String(b) ? 1 : -1) {
```

Comment:
answer to your question can be something like that)
defaultCompareFuncion create above applyCustomSort

```
const defaultCompareFunction = (a, b) => {
  const strA = String(a);
  const strB = String(b);

  switch (true) {
    case strA > strB:
      return 1;
    case strA < strB:
      return -1;
    default:
      return 0;
  }
};
```
```suggestion
  [].__proto__.sort2 = function(compareFunction = defaultCompareFunction) {
```

Reply:
Hi) I know, how to create separate method, sorry for this confuse. I can't understand how this method use as default for compareFunction. I can't just add it as parameter, it's not working( 

`[].__proto__.sort2 = function(compareFunction = defaultCompareFunction ) {`

Reply:
It's absolutely clear now, maybe even too much) Thank you!